src/store/reducers/cart.ts

```diff
@@ -3,10 +3,18 @@ import { AppActions } from '../../actions';
 import { CartTypes } from '../../types/cart';
 import { Product } from '../../types/product';
 
+const sortByTitle = (prod1: Product, prod2: Product) => {
```

Comment:
This definitely gets the job done! Another option is to use [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare), to make it a one-liner.

Reply:
me leveling up!